scss/_pagination.scss

```diff
@@ -78,10 +78,6 @@
 }
 
 .page-item {
-  &:not(:first-child) .page-link {
```

Comment:
IMO, this should be kept as is since it comes directly from Bootstrap, but you could set `$pagination-margin-start` to 0

Reply:
Ok, for my information, and next time, how can I know that it comes directly from Boosted please ?

Reply:
We used to add `// Boosted mod` at the end of the lines when the modifications only affects few lines.
Otherwise, we used to add `// Boosted mod` and `// End mod` that encapsulate lines of code if it affects several lines.